taworks/taform/views.py

```diff
@@ -212,10 +212,19 @@ def apply(request):
                 }
         try:
             studentID=str(request.POST['student_id'])
-            if len(studentID) > 8:
+            studentID_length=str(request.POST['student_id'])
+            apps_made= models.Application.objects.filter(student__student_id=studentID, preference__in = [1,2,3]).count()
+            app_id = 0
+            print apps_made
```

Comment:
remove

Reply:
I don't think you need line 215-223.

just need to initialize app_id = None somewhere.

Reply:
can you explain why you think I don't need lines 215 to 223?